packages/dashboard-provider/manual-test/metacoin-truffle/truffle-config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  networks: {
+    browser: {
+      url: "http://localhost:5000/rpc",
+      network_id: "*",
+      skipDryRun: true,
+      networkCheckTimeout: 120000
+    }
+  }
+};
```

Comment:
Maybe the provider could alert the user to add/switch MM to the truffle develop network along with further needed instructions.

Reply:
Watching the reaction from my demo yesterday, I think `truffle dashboard` is going to make quite a big splash by itself.

Still want to see this, but I think we can wait until after initial release and prioritize alongside other enhancements.

Reply:
Would definitely be cool to have this. At the same time, when using `truffle develop` it's much less interesting to use the Truffle Dashboard in its current state. Right now the main use of the dashboard is to deploy your contracts with metamask so you don't have to copy-paste your mnemonic. With `truffle develop` that is much less relevant since you ca just use the `truffle develop` mnemonic for deployments.

In later stages of the dashboard, like with the visual debugger I think it makes a lot more sense to have this `truffle develop` integration. So I agree that we should hold off on this for now, and revisit at that point.